neo/Network/P2P/Payloads/MinerTransaction.cs

```diff
@@ -8,8 +8,6 @@ namespace Neo.Network.P2P.Payloads
 {
     public class MinerTransaction : Transaction
     {
-        public uint Nonce;
```

Comment:
Isn't this useful here to provide randomization of Miners tx?

Reply:
now is on all TX, so is reused